packages/web-components/src/components/va-notification/va-notification.tsx

```diff
@@ -116,7 +116,7 @@ export class VaNotification {
           <div class={classes} role="alert">
             <i aria-hidden="true" role="img" class={symbol}></i>
             <div class="body" role="presentation">
-              {HeadlineLevel ? <HeadlineLevel part="headline">{headline}</HeadlineLevel> : headline}
+              {HeadlineLevel ? <HeadlineLevel part="headline">{headline}</HeadlineLevel> : null}
```

Comment:
This needs to be `{headline ?` not `{HeadlineLevel`, since HeadlineLevel will always resolve to 'something' but `headline` is not guaranteed to be non-null.